scripts/start_functions.R

```diff
@@ -589,3 +589,35 @@ getReportData <- function(path_to_report_bioenergy, mute_ghgprices_until = "y201
     .emissionPrices(ghgmag, mute_ghgprices_until)
   }
 }
+
+# Will not actually solve the model: after compilation, this just copies the results
+# of a previous run, useful for testing compilation and input/output handling.
+# Used in scripts/start/extra/empty_model.R and tests for REMIND-MAgPIE coupling.
+configureEmptyModel <- function(cfg, inputGdxPath) {
+    message("Configuring to use empty MAgPIE model, reproduces prior run ", inputGdxPath)
+    originalModel <- file(cfg$model, "r")
```

Comment:
```suggestion
    originalModel <- withr::local_connection(file(cfg$model, "r"))
```
This ensures the file handle is properly closed when the function exits.